### ISSUE TITLE
Editable form not in popover but visible in gridview

### ISSUE THREAD

**Issue by drsdre** [kartik-v/yii2-grid#435]
The Editable functions since 3.0.9 has started printing the form outside popover-content and is now visible in all the fields.

The rendered HTML code of a editable field:
```` html
<div id="brand-0-status-cont" class="kv-edcol-3502a3e3 kv-editable"><div class="kv-editable-content"><form id="w9" class="kv-editable-form" action="/brand/index" method="post">
<input type="hidden" name="_csrf" value="WFFtcngudfdfdffdfdQFs6OzAfFwMTVFQuNGEmMFVCXi40ECZAE1csCg=="><input type="hidden" name="hasEditable" value="0">
<input type="hidden" name="editableIndex" value="0"><input type="hidden" name="editableKey" value="9"><input type="hidden" name="editableAttribute" value="status">
<div class="kv-editable-parent form-group field-brand-0-status required">
<input type="hidden" name="Brand[0][status]" value="">
<div id="brand-0-status" class="kv-editable-input form-control">
<label><input type="radio" name="Brand[0][status]" value="0" checked=""> Active</label>
<label><input type="radio" name="Brand[0][status]" value="1"> Paused</label>
</div>
<div class="help-block"></div>
</div>
</form></div>
<button type="button" id="brand-0-status-targ" class="kv-editable-value kv-editable-link" data-toggle="popover-x" data-placement="right" data-target="#brand-0-status-popover">Active</button>
<div id="brand-0-status-popover" class="kv-editable-popover skip-export has-footer popover popover-default" role="dialog">
<div class="arrow"></div><div class="popover-title">
<button type="button" class="close" data-dismiss="popover-x" aria-hidden="true">×</button>
<i class="glyphicon glyphicon-edit"></i> Edit Status
</div>
<div class="popover-content">
</div>
<div class="popover-footer">
<div class="kv-editable-loading" style="display:none">&nbsp;</div><button type="button" class="btn btn-sm btn-default kv-editable-reset" title="Reset"><i class="glyphicon glyphicon-ban-circle"></i></button><button type="button" class="btn btn-sm btn-primary kv-editable-submit" title="Apply"><i class="glyphicon glyphicon-save"></i></button>
</div>
</div></div>
````

Column definition for GridView:
```` php
	[
		'class' => 'kartik\grid\EditableColumn',
		'attribute' => 'status',
		'value' => function ($data) {
			return $data->statusName;
		},
		'filter' => $searchModel::$statuses,
		'editableOptions'=> function ($model, $key, $index, $widget) {
			return [
				'header' => 'Status',
				'name'=>'status',
				'inputType' => Editable::INPUT_RADIO_LIST,
				'data'=> \common\models\Feed::$statuses,
			];
		},
	],
````

**Comment by drsdre** [comment]
Reverted back Yii2-grid to 3.0.8 which didn't fix the problem.
Rolled forward again to 3.0.9 and rolled back kartik-v/yii2-popover-x to 1.3.1 which fixed problem.
My guess is that something internally in kartik-v/yii2-popover-x broke.

Please close this bug.

**Comment by kartik-v** [comment]
Resolved via kartik-v/yii2-popover#13
